coursier.bzl

```diff
@@ -22,16 +22,25 @@ load(
     "COURSIER_CLI_BAZEL_MIRROR_URL",
     "COURSIER_CLI_GITHUB_ASSET_URL",
     "COURSIER_CLI_SHA256",
+    "JQ_VERSIONS",
 )
 
 _BUILD = """
 package(default_visibility = ["//visibility:{visibility}"])
 
-exports_files(["pin"])
-
 load("@rules_jvm_external//private/rules:jvm_import.bzl", "jvm_import")
 load("@rules_jvm_external//private/rules:jetifier.bzl", "jetify_aar_import", "jetify_jvm_import")
 
+sh_binary(
```

Comment:
this cases `bazel build @maven//...` to fail:

```
ERROR: missing input file 'external/maven/pin.sh', owner: '@maven//:pin.sh'
ERROR: /private/var/tmp/_bazel_jingwen/5e64b778ca5f4ff8a6ca994b3f33d8f8/external/maven/BUILD:7:1: @maven//:pin: missing input file 'LabelCause{label=@maven//:pin.sh, msg=missing input file 'external/maven/pin.sh', owner: '@maven//:pin.sh'}'
ERROR: /private/var/tmp/_bazel_jingwen/5e64b778ca5f4ff8a6ca994b3f33d8f8/external/maven/BUILD:7:1 1 input file(s) do not exist
```

This is because we don't expect the pinned repo to contain a `pin.sh` file, since pinning is done on the unpinned repo. Do you think you can find a way to continue making `bazel build @maven//...` work?

Reply:
builds now.